### PR TITLE
Add new filters for projects list

### PR DESCRIPTION
**Pull request by maxceem**
winning submission for challenge 30083089 - Topcoder Connect - Filter project table by columns, fix for issue #245

The quality of the submission looks great to me.

### CODE REVIEW

File: migrations/elasticsearch_sync.js [hunk 1]
```diff
@@ -209,7 +209,8 @@ function getRequestBody(indexName) {
         },
       },
       id: {
-        type: 'long',
+        type: 'string',
+        index: 'not_analyzed',
```
Comment (maxceem) [inline migrations/elasticsearch_sync.js]: @vikasrohit this change has been done to the ES sync script which looks like required to filter by substring in `id`.

I don't see any drawbacks of this change. But maybe you have any thoughts on this.

**Comment by maxceem** [comment]
Accidently, removed and after restored the branch.

File: migrations/elasticsearch_sync.js (hunk 1 above)
Comment (RishiRajSahu) [reply migrations/elasticsearch_sync.js]: @maxceem can we separate out this id substring change from this PR if we have to reindex all projects again in order to make the filtering by ID substring enable ?

**Comment by maxceem** [comment]
@RishiRajSahu I'm closing this PR as I've created another one https://github.com/topcoder-platform/tc-project-service/pull/256 which doesn't have an update for ID.

**Pull request closed without merging**